### YamlMime:ManagedReference
items:
- uid: System.ExecutionEngineException
  id: ExecutionEngineException
  children:
  - System.ExecutionEngineException.#ctor
  - System.ExecutionEngineException.#ctor(System.String)
  - System.ExecutionEngineException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ExecutionEngineException
  nameWithType: ExecutionEngineException
  fullName: System.ExecutionEngineException
  type: Class
  summary: "Eccezione generata quando si verifica un errore interno nel motore di esecuzione di common language runtime. Questa classe non può essere ereditata."
  remarks: "Non sussiste alcuna alternativa non obsoleta per ExecutionEngineException. Se un&quot;ulteriore esecuzione dell&quot;applicazione non può essere subito, utilizzare il <xref:System.Environment.FailFast%2A>metodo.</xref:System.Environment.FailFast%2A>"
  syntax:
    content: >-
      [System.Obsolete("This type previously indicated an unspecified fatal error in the runtime. The runtime no longer raises this exception so this type is obsolete.")]

      [System.Runtime.InteropServices.ComVisible(true)]

      public sealed class ExecutionEngineException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.ExecutionEngineException.#ctor
  id: '#ctor'
  parent: System.ExecutionEngineException
  langs:
  - csharp
  name: ExecutionEngineException()
  nameWithType: ExecutionEngineException.ExecutionEngineException()
  fullName: System.ExecutionEngineException.ExecutionEngineException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.ExecutionEngineException&quot;> </xref> classe."
  remarks: "Questo costruttore inizializza la <xref:System.Exception.Message%2A>proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l&quot;errore, ad esempio &quot;si è verificato un errore interno.&quot;</xref:System.Exception.Message%2A> Questo messaggio prende in considerazione le impostazioni cultura correnti.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un&quot;istanza di <xref:System.ExecutionEngineException>.</xref:System.ExecutionEngineException>      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Un riferimento null (`Nothing` in Visual Basic). |   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore localizzato. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ExecutionEngineException ();
    parameters: []
  overload: System.ExecutionEngineException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ExecutionEngineException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.ExecutionEngineException
  langs:
  - csharp
  name: ExecutionEngineException(String)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.ExecutionEngineException&quot;> </xref> classe con un messaggio di errore specificato."
  remarks: "Questo costruttore inizializza la <xref:System.Exception.Message%2A?displayProperty=fullName>proprietà della nuova eccezione utilizzando il `message` parametro.</xref:System.Exception.Message%2A?displayProperty=fullName> Il contenuto di `message` deve essere facilmente comprensibile. Il chiamante del costruttore è necessario per garantire che la stringa sia stata localizzata per la lingua corrente.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un&quot;istanza di <xref:System.ExecutionEngineException>.</xref:System.ExecutionEngineException>      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Un riferimento null (`Nothing` in Visual Basic). |   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ExecutionEngineException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio che descrive l&quot;errore."
  overload: System.ExecutionEngineException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.ExecutionEngineException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.ExecutionEngineException
  langs:
  - csharp
  name: ExecutionEngineException(String,Exception)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String,Exception)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.ExecutionEngineException&quot;> </xref> classe con un messaggio di errore specificato e un riferimento all&quot;eccezione interna che è la causa dell&quot;eccezione corrente."
  remarks: "Un&quot;eccezione generata come risultato diretto di un&quot;eccezione precedente deve includere un riferimento all&quot;eccezione precedente nella <xref:System.Exception.InnerException%2A>proprietà.</xref:System.Exception.InnerException%2A> Il <xref:System.Exception.InnerException%2A>proprietà restituisce lo stesso valore passato al costruttore o un riferimento null (`Nothing` in Visual Basic) se il <xref:System.Exception.InnerException%2A>proprietà non fornisce il valore di eccezione interna al costruttore.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un&quot;istanza di <xref:System.ExecutionEngineException>.</xref:System.ExecutionEngineException>      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Il riferimento all&quot;eccezione interna. |   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public ExecutionEngineException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio di errore che spiega il motivo dell&quot;eccezione."
    - id: innerException
      type: System.Exception
      description: "Eccezione che è la causa dell&quot;eccezione corrente. Se il `innerException` parametro non è un riferimento null (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> in Visual Basic), l&quot;eccezione corrente viene generata in un <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blocco che gestisce l&quot;eccezione interna."
  overload: System.ExecutionEngineException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.ExecutionEngineException.#ctor
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException()
  nameWithType: ExecutionEngineException.ExecutionEngineException()
  fullName: System.ExecutionEngineException.ExecutionEngineException()
- uid: System.ExecutionEngineException.#ctor(System.String)
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException(String)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.ExecutionEngineException.#ctor(System.String,System.Exception)
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException(String,Exception)
  nameWithType: ExecutionEngineException.ExecutionEngineException(String,Exception)
  fullName: System.ExecutionEngineException.ExecutionEngineException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.ExecutionEngineException.#ctor*
  parent: System.ExecutionEngineException
  isExternal: false
  name: ExecutionEngineException
  nameWithType: ExecutionEngineException.ExecutionEngineException
